hmda/src/main/scala/hmda/persistence/institution/InstitutionState.scala

```diff
@@ -19,6 +19,7 @@ case class InstitutionState(institution: Option[Institution],
         } else {
           InstitutionState(this.institution, filing :: filings)
         }
+      case _ => this
```

Comment:
Should we be swallowing all other match cases or logging that they occurred with an event?